otter/log/spec.py

```diff
@@ -6,12 +6,12 @@
 
 from toolz.curried import assoc
 from toolz.dicttoolz import keyfilter
-from toolz.functoolz import compose
+from toolz.functoolz import compose, curry
 
 from twisted.python.failure import Failure
 
 
-_json_len = compose(len, json.dumps)
+_json_len = compose(len, curry(json.dumps, default=repr))
```

Comment:
How about using `curry(json.dumps, cls=LoggingEncoder)` instead of just `default=repr`? that should be the best guarantee that it'll match our actual encoding behavior